### PR TITLE
Automl

### PR DESCRIPTION
**Pull request by nirupa-kumar**
Missing Create and Delete Model tests.

### CODE REVIEW

**Comment by nirupa-kumar** [comment]
@nnegrey Please review. Let me know if someone else needs to be reviewing this. I will be adding the others AutoMLs as well.

File: vision/automl/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# AutoML Sample
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=vision/beta/cloud-client/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Google Cloud Vision API][vision] provides feature detection for images.
```
Comment (tigersoldier) [inline vision/automl/README.md]: This summery should be for AutoML Vision, not Vision API in general

File: language/automl/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# AutoML Sample
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=vision/beta/cloud-client/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Google Cloud Natural Language API][language] provides feature detection for images.
```
Comment (tigersoldier) [inline language/automl/README.md]: This API is for AutoML Natural Language, not for the Natural Language API in general.

Also, it's not for detecting images

File: language/automl/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+This API is part of the larger collection of Cloud Machine Learning APIs.
+
+This sample Java application demonstrates how to access the Cloud Natural Language AutoML API
+using the [Google Cloud Client Library for Java][google-cloud-java].
+
+[language]: https://cloud.google.com/language/docs/
```
Comment (tigersoldier) [inline language/automl/README.md]: Should be https://cloud.google.com/language/automl/docs/

File: language/automl/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+#### Get dataset
+```
+mvn exec:java -Dexec.mainClass="com.google.cloud.language.samples.DatasetApi" -Dexec.args="get_dataset [dataset-id]"
+```
+
+#### Import data
+```
+mvn exec:java -Dexec.mainClass="com.google.cloud.language.samples.DatasetApi" -Dexec.args="import_data gs://java-docs-samples-testing/happiness.csv"
```
Comment (tigersoldier) [inline language/automl/README.md]: Missing [dataset-id] in args

File: translate/automl/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# AutoML Translate Sample
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=vision/beta/cloud-client/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Google Cloud Translate API][translate] provides feature AutoML.
```
Comment (tigersoldier) [inline translate/automl/README.md]: Should be AutoML Translation

File: translate/automl/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+This API is part of the larger collection of Cloud Machine Learning APIs.
+
+This sample Java application demonstrates how to access the Cloud Translate AutoML API
+using the [Google Cloud Client Library for Java][google-cloud-java].
+
```
Comment (tigersoldier) [inline translate/automl/README.md]: Missing URLs for [translate] and [google-cloud-java]

File: translate/automl/README.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+#### Get dataset
+```
+mvn exec:java -Dexec.mainClass="com.google.cloud.translate.samples.DatasetApi" -Dexec.args="get_dataset [dataset-id]"
+```
+
+#### Import data
+```
+mvn exec:java -Dexec.mainClass="com.google.cloud.translate.samples.DatasetApi" -Dexec.args="import_data gs://java-docs-samples-testing/en-ja.csv"
```
Comment (tigersoldier) [inline translate/automl/README.md]: Missing [dataset-id] in args

File: vision/automl/README.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+This API is part of the larger collection of Cloud Machine Learning APIs.
+
+This sample Java application demonstrates how to access the Cloud Vision API
+using the [Google Cloud Client Library for Java][google-cloud-java].
+
+[vision]: https://cloud.google.com/vision/docs/
```
Comment (tigersoldier) [inline vision/automl/README.md]: The URL should be 

https://cloud.google.com/vision/automl/docs/

File: language/automl/src/test/java/com/google/cloud/language/samples/DatasetApiIT.java [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for Automl natural language "Dataset API" sample. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class DatasetApiIT {
+
+  private static final String PROJECT_ID = "java-docs-samples-testing";
+  private static final String BUCKET = PROJECT_ID + "-vcm";
```
Comment (sirtorry) [inline language/automl/src/test/java/com/google/cloud/language/samples/DatasetApiIT.java]: vcm should be lcm for language

File: translate/automl/src/test/java/com/google/cloud/translate/automl/DatasetApiIT.java [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for Automl translation "Dataset API" sample. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class DatasetApiIT {
+
+  private static final String PROJECT_ID = "java-docs-samples-testing";
+  private static final String BUCKET = PROJECT_ID + "-vcm";
```
Comment (sirtorry) [inline translate/automl/src/test/java/com/google/cloud/translate/automl/DatasetApiIT.java]: vcm should be tcm for translate.

**Pull request merged**